service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -104,6 +112,41 @@ void aboutToStartCallbackShouldSplitDirectionsIntoSeparateCollections() throws E
         assertThat(extractDirections(caseData.getCourtDirections())).containsOnly(directions.get(5));
     }
 
+    @Nested
+    class StateChangeTests {
+        private final String event = "internal-changeState:Gatekeeping->SDO_READY_TO_SEND";
```

Comment:
might be a good idea to extract this to a case state enum in future. Not required here